### PR TITLE
Aisulu intro

### PR DESCRIPTION
**Pull request by AisOmar**
Added me to CVV Website content repo

### CODE REVIEW

**Review by hollandjg — COMMENTED** [review]
Looks good, just one change I'd make to the team/subteam

File: about/people.yml [hunk 1]
```diff
@@ -314,3 +314,12 @@ data:
     brown_directory_uuid: 504c3db1-c3ec-4b15-9350-704e8bbe3036
     bio: Carlos is a data scientist at the CCV. Previously a Mathematics tenure-track faculty member at universities in the US and the Dominican Republic, Carlos likes using his skills working with data for wider social impact.
     image: carlos_main.jpg
+  - name: Aisulu Omar
+    type: Full Time
+     team: Advanced Research Computing
+     subteam: Data & Visualization
```
Comment (hollandjg) [inline about/people.yml:320]:
```suggestion
    subteam: Data & Visualization
```

File: about/people.yml [hunk 2]
```diff
@@ -314,3 +314,12 @@ data:
     brown_directory_uuid: 504c3db1-c3ec-4b15-9350-704e8bbe3036
     bio: Carlos is a data scientist at the CCV. Previously a Mathematics tenure-track faculty member at universities in the US and the Dominican Republic, Carlos likes using his skills working with data for wider social impact.
     image: carlos_main.jpg
+  - name: Aisulu Omar
+    type: Full Time
+     team: Advanced Research Computing
```
Comment (hollandjg) [inline about/people.yml:319]:
```suggestion
    team: Advanced Research Computing
```

**Review by mirestrepo — COMMENTED** [review]
The alignment seems off

**Review by mirestrepo — APPROVED** [review]
You are first on the people section!

**Review by hollandjg — APPROVED** [review]
Looks good to me!

**Pull request merged**